mod/05_email/set/right/bcc.rb

```diff
@@ -18,7 +22,7 @@ def process_email_addresses context_card, format_args, args
 format :email_text do
   view :email_addresses do |args|
     context = args[:context] || self
-    card.item_names.map do |item_name|
+    card.item_names(:context=>context.cardname).map do |item_name|
```

Comment:
I did that when I merged.  It's ok not to send PR's for comments, right? ;)

On Thu, Feb 12, 2015 at 6:52 AM, Gerry Gleason <notifications@github.com>
wrote:

> In mod/05_email/set/right/bcc.rb
> <https://github.com/wagn/wagn/pull/302#discussion_r24581360>:
>
> > @@ -18,7 +22,7 @@ def process_email_addresses context_card, format_args, args
> >  format :email_text do
> >    view :email_addresses do |args|
> >      context = args[:context] || self
> > -    card.item_names.map do |item_name|
> > +    card.item_names(:context=>context.cardname).map do |item_name|
>
> Agreed. This can be fixed so it doesn't need that line, but fine for now.
> Maybe drop in a TODO comment.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wagn/wagn/pull/302/files#r24581360>.
>



-- 
Ethan McCutchen
One of the Wagneers, Wagn.org

Wagn. How pioneers roll.

s: ethan.mccutchen
t: @intogreater